src/buffer.c

```diff
@@ -24,8 +24,8 @@
 #include <stdint.h>
 #include <string.h>
 
-
-size_t BUFFER_SIZE     = 512;
+// BUFFER_SIZE is set to 4k
```

Comment:
"size_t BUFFER_SIZE = 4096;" 

stands pretty well on its own and does not need a comment.

Reply:
Removed comment, thanks